src/widget/wcuemenupopup.h

```diff
@@ -30,10 +31,9 @@ class WCueMenuPopup : public QWidget {
         }
     }
 
-    void popup(const QPoint& p, QAction* atAction = nullptr) {
-        Q_UNUSED(atAction);
-        qDebug() << "Showing menu at" << p;
-        move(p);
+    void popup(const QPoint& p) {
+        auto parentWidget = dynamic_cast<QWidget*>(parent());
```

Comment:
A static_cast is sufficient here since we pass a QWidget in the constructor. For additional safety the dynamic_cast could be used in a debug assertion.